.travis.yml

```diff
@@ -356,12 +356,12 @@ before_install:
  - if [ "$TASK" == "spellintian" -o "$TASK" == "spellintian-duplicates" ]; then wget "http://old-releases.ubuntu.com/ubuntu/pool/main/l/lintian/lintian_2.5.104_all.deb"; sudo dpkg -i lintian_*.deb; sudo apt-get install -f -y; fi # Install a later lintian
 
 after_failure:
-# Disabled as otherwise the logfile is too big
-#  - if [ -f ${TRAVIS_BUILD_DIR}/config.log ]; then cat ${TRAVIS_BUILD_DIR}/config.log; fi
-#  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/config.log; fi
-#  - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/config.log; fi
   - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/test-suite.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/test-suite.log; fi
   - if [ -f ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/test-suite.log ]; then cat ${TRAVIS_BUILD_DIR}/ola-*/_build/sub/test-suite.log; fi
+# Disabled as otherwise the logfile is too big
+#  - if [ -f ${TRAVIS_BUILD_DIR}/config.log ]; then cat ${TRAVIS_BUILD_DIR}/config.log; fi
```

Comment:
I think this one can be left uncommented too now, just in case it exists somewhere.

Reply:
doesn't hurt.  I need to push a change to the pycompiletest anyway b/c I forgot -f breaks the mac build

Reply:
Ah, how so?

Reply:
-f forces it to recompile all of the python code regardless of timestamp etc.  I think the problem is that with it trying to run the tests in parallel it winds up trying to delete files while they're being used for other tests and that causes permission denied errors.   For a clean build there really isn't ever a reason to use -f, so I think it's fine without it.  On linux I guess the OS is OK with deleting a file and leaving it open by another process, so it doesn't cause a problem.

Reply:
Ah yes the parallel build! We still run the java stuff serially for a different reason but I don't think it's worth slowing down all the other python stuff just to fix this, as you say dropping -f seems good enough.